ui-core/js/services/engine.ts

```diff
@@ -433,6 +432,13 @@ function initializeWithAuthorization(callback, tenantId, flowId, flowVersionId,
 
                 }
 
+                await Component.addCustomComponents({
```

Comment:
@TomFullalove Does this need to be awaited? I think its fire and forget

Reply:
No, it doesn't need to be now.

I used to have an approach to "pause everything" until the components are added,
but I think this "fire and forget" method is better.

I've removed the async.